src/main/java/com/synopsys/integration/alert/web/config/FieldValidationAction.java

```diff
@@ -63,6 +63,9 @@ public void validateConfig(Map<String, ConfigField> descriptorFields, FieldModel
                 if (validationResult.hasErrors()) {
                     fieldErrors.put(key, validationResult.combineErrorMessages());
                 }
+                if (validationResult.hasWarnings()) {
+                    fieldErrors.put(key, validationResult.combineWarningMessages());
```

Comment:
Should we be adding warnings to the fieldErrors?

Reply:
This is a work in progress.  We are keeping the same functionality for now and making sure that the code is in place to be able to add warnings even though they are treated as errors. The next branch and refactoring will to convert to the objects that contain information about warnings or errors. Taking a phased approach to this refactor.

Reply:
That's our next step.   We know this code works with the existing code and provides the pieces to add warnings.  Next step is to incorporate all of it together.

Reply:
Can we add a TODO in the places that we know need to change?

Reply:
We will add TODOs as we move forward.  We are working on a feature branch so the feature branch will ultimately be reviewed by everyone.